app/static/src/app/components/help/DraggableDialog.vue

```diff
@@ -1,21 +1,28 @@
 <template>
   <div ref="draggable" class="draggable-dialog p-2" :style="dialogStyle">
-    <div class="dragtarget" @mousedown="handleDragStart" @touchstart="handleDragStart">
+    <div class="dragtarget clearfix" @mousedown="handleDragStart" @touchstart="handleDragStart">
       <h3 class="move prevent-select">
          <vue-feather type="move" class="grey"></vue-feather>
-          {{title}}
-        <vue-feather type="x" class="clickable grey float-end" @click="close"></vue-feather>
+         <vue-feather type="x" class="clickable grey float-end" @click="close"></vue-feather>
+         <div class="dialog-title">{{title}}</div>
       </h3>
     </div>
     <div class="overflow-auto draggable-content">
       <slot></slot>
     </div>
+    <svg class="resize-handle" height="24" width="24" @mousedown="handleResizeStart" @touchstart="handleResizeStart">
+      <polygon points="0,24 24,0 24,24" style="fill:white" />
```

Comment:
I bet this was fun to write without hot reloading. I like the idea - tbh I am surprised that there's no up-left/down right version of [this icon](https://fontawesome.com/icons/arrow-up-right-and-arrow-down-left-from-center?s=solid&f=sharp) but yours takes up less space in any case.

I think it looks slightly off with the extra lower border because the corner on the internal rectangle is not centred on the handle, I think that it would be fine not to have a larger lower border

![image](https://user-images.githubusercontent.com/1558093/218453378-a5f88d79-c313-4562-8480-bd24c1fccd27.png)

Reply:
Yeah, the reason I did that was so it didn't overlap the code area - but I think you're right that it looks neater with the same margin at the bottom.